reana_server/rest/info.py

```diff
@@ -120,8 +125,20 @@ def info(user, **kwargs):  # noqa
                 title="List of supported compute backends",
                 value=SUPPORTED_COMPUTE_BACKENDS,
             ),
+            kubernetes_memory_limit=dict(
+                title="Default memory limit for Kubernetes backend jobs",
+                value=REANA_KUBERNETES_JOBS_MEMORY_LIMIT,
+            ),
+            kubernetes_max_memory_limit=dict(
+                title="Maximum allowed memory limit for Kubernetes backend jobs",
+                value=REANA_KUBERNETES_JOBS_MAX_USER_MEMORY_LIMIT,
+            ),
+            default_workspace_retention_period=dict(
+                title="Default number of days to retain workspace files",
+                value=WORKSPACE_RETENTION_PERIOD,
+            ),
         )
-        return InfoSchema().dump(info)
+        return InfoSchema().dump(cluster_information)
```

Comment:
The OpenAPI specification needs to be updated to include the changes to the response